release-controller/release_notes.py

```diff
@@ -10,7 +10,7 @@
 import typing
 from dataclasses import dataclass
 from git_repo import GitRepo
-from util import bazel_binary
+from commit_annotator import GUESTOS_CHANGED_NOTES_NAMESPACE
```

Comment:
Is commit_annotator an actual top level module???

Reply:
no, but if you wish to refactor this to work properly, be my guest.